Description: Update docs
  Added information on dmalloc_tag and posix_memalign to texinfo file
 .
 dmalloc (5.5.2-3) unstable; urgency=low
 .
    *Document dmalloc tag and posix_memalign
Author: Johann Klammer <klammerj@a1.net>

--- dmalloc-5.5.2.orig/docs/dmalloc.texi
+++ dmalloc-5.5.2/docs/dmalloc.texi
@@ -870,7 +870,7 @@ checking the heap after it sees a call f
 @cindex dmalloc.h file
 
 By including @file{dmalloc.h} in your C files, your calls to malloc,
-calloc, realloc, recalloc, memalign, valloc, strdup, and free are
+calloc, realloc, recalloc, memalign, posix_memalign, valloc, strdup, and free are
 replaced with calls to _dmalloc_malloc, _dmalloc_realloc, and
 _dmalloc_free with various flags.  Additionally the library replaces
 calls to xmalloc, xcalloc, xrealloc, xrecalloc, xmemalign, xvalloc,
@@ -890,6 +890,7 @@ This line from a log file shows that mem
 
 @cindex recalloc
 @cindex memalign
+@cindex posix_memalign
 @cindex valloc
 @cindex strdup
 
@@ -1252,6 +1253,18 @@ not all other malloc libraries.
 
 @c --------------------------------
 
+@cindex dmalloc_tag function
+@cindex tag a pointer
+
+@deftypefun int dmalloc_tag ( const DMALLOC_PNT @var{pnt}, char *@var{file},int @var{line})
+
+This function sets a pointer's file and line information to the supplied values.
+@var{file} @emph{must} be a constant string, it is not copied.
+
+@end deftypefun
+
+@c --------------------------------
+
 @cindex dmalloc_track function
 @cindex track memory calls
 
